.github/CODEOWNERS

```
@@ -9,7 +9,20 @@
 # The docs
 /cloud-account/                             @elastic/admin-docs
 /deploy-manage/                             @elastic/admin-docs
+
 /explore-analyze/                           @elastic/core-docs @elastic/experience-docs
+/explore-analyze/alerts-cases/              @elastic/experience-docs
+/explore-analyze/dashboards/                @elastic/experience-docs
+/explore-analyze/discover/                  @elastic/experience-docs
+/explore-analyze/elastic-inference/         @elastic/developer-docs
```

```suggestion
/explore-analyze/elastic-inference/         @elastic/developer-docs @elastic/experience-docs
```
Will impact experience too